src/main/java/com/softserve/academy/spaced/repetition/controller/DeckCommentController.java

```diff
@@ -25,74 +28,73 @@
 import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
 
 @RestController
+@RequestMapping("/api/decks/{deckId}/")
 public class DeckCommentController {
+
     private static final Logger LOGGER = LoggerFactory.getLogger(DeckCommentController.class);
 
     @Autowired
     private DeckCommentService commentService;
 
     @Auditable(action = AuditingAction.VIEW_ALL_COMMENTS_FOR_DECK)
-    @GetMapping(value = "/api/category/{categoryId}/deck/{deckId}/comments")
-    public ResponseEntity<List<CommentDTO>> getAllCommentsForDeck(@PathVariable Long categoryId,
-                                                                  @PathVariable Long deckId) {
+    @GetMapping(value = "comments")
+    @PreAuthorize("hasPermission('DeckComment','READ')")
```

Comment:
Would be good make as const and move these 'DeckComment' and 'READ' into separate class